enterprise/cypher/morsel-runtime/src/test/scala/org/neo4j/cypher/internal/runtime/vectorized/operators/MergeSortOperatorTest.scala

```diff
@@ -49,6 +51,23 @@ class MergeSortOperatorTest extends CypherFunSuite {
     out.longs should equal(longs)
   }
 
+  test("top on a single morsel") {
+    val slot = LongSlot(0, nullable = false, CTNode)
+    val columnOrdering = Seq(Ascending(slot))
+    val info = new SlotConfiguration(mutable.Map("apa" -> slot), 1, 0)
```

Comment:
Creating slot-configurations manually tends to be nicer by doing
```
SlotConfiguration.empty
      .newLong("apa", false, CTNode)
```

instead. This also makes sure the configuration is internally consistent.

Reply:
Good to know. In this case I think it is somewhat confusing though, since I still need the slot for the `columnOrdering`

Reply:
You can get it out the SlotConfiguration with apply, e.g. `slots("apa")`